java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -992,6 +1068,24 @@ public final ColumnBuilder append(boolean value) {
       return this;
     }
 
+    public final ColumnBuilder append(BigDecimal value) {
+      growBuffersAndRows(false, currentIndex * type.getSizeInBytes() + type.getSizeInBytes());
+      assert type.isDecimalType();
```

Comment:
This assert is unnecessary, the code properly throws if the type is not one of the expected decimal types.

Reply:
Removed.